tools/c2chapel/c2chapel.py

```diff
@@ -343,6 +343,12 @@ def genVar(decl):
     print("extern var " + name + " : " + ty + ";")
     print()
 
+
+def genEnum(decl):
+    if type(decl) == c_ast.Enum:
+        for val in decl.values.enumerators:
```

Comment:
Let's add a generated comment before the constants. If the enum has a name, we can print it there, otherwise we can just call it "anonymous"